data/src/main/java/com/terning/data/service/InternService.kt

```diff
@@ -0,0 +1,13 @@
+package com.terning.data.service
+
+import com.terning.data.dto.BaseResponse
+import com.terning.data.dto.response.InternResponseDto
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface InternService {
+    @GET("api/v1/announcements/{internshipAnnouncementId}")
+    suspend fun getInternInfo(
+        @Path("internshipAnnouncementId") internshipAnnouncementId: Int,
+    ): BaseResponse<InternResponseDto>
```

Comment:
```suggestion
    @GET("api/v1/announcements/{internshipAnnouncementId}")
    suspend fun getInternInfo(
        @Path("internshipAnnouncementId")
         internshipAnnouncementId: Int,
    ): BaseResponse<InternResponseDto>
```
이렇게 해주면 코드가 조금 더 이뻐질 것 같네용